File: app/src/main/java/com/example/milen/hope/ActivityList.java
package com.example.milen.hope;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class ActivityList extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_list);
    }

    public void onCreateButtonClicked(View view)
    {
        Intent intent = new Intent(this, CreateList.class);
        startActivity(intent);
    }

    public void onSeeListsButtonClicked(View view)
    {
        Intent intent = new Intent(this, SeeLists.class);
        startActivity(intent);
    }
}

File: app/src/main/java/com/example/milen/hope/ActivityList.kt
package com.example.milen.hope

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class ActivityList : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_list)
    }

    fun onCreateButtonClicked(view: View) {
        val intent = Intent(this, CreateList::class.java)
        startActivity(intent)
    }

    fun onSeeListsButtonClicked(view: View) {
        val intent = Intent(this, SeeLists::class.java)
        startActivity(intent)
    }
}
